benchmark/streamly-benchmarks.cabal

```diff
@@ -113,13 +118,24 @@ common bench-depends
     -- other libraries
     , streamly            >= 0.7.0
     , random              >= 1.0   && < 2.0
-    , gauge               >= 0.2.4 && < 0.3
     , transformers        >= 0.4   && < 0.6
     , containers          >= 0.5   && < 0.7
     , typed-process     >= 0.2.3 && < 0.3
     , directory         >= 1.2.2 && < 1.4
     , ghc-prim          >= 0.4   && < 0.8
 
+  if flag(use-gauge)
+    cpp-options:    -DGUAGE
+    build-depends:  gauge >= 0.2.4 && < 0.3
+  else
+    cpp-options:    -DTASTY
```

Comment:
We do not need `-DTASTY`, we can check for `#if !defined(USE_GAUGE)` or use the else case of `#ifdef USE_GAUGE`.

Reply:
Actually it's enough to check that `MIN_VERSION_gauge` is defined instead of defining `GAUGE` manually.

Reply:
Yeah, that should do.